ideas/dropdown.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+// example for one button
+
+function handleNavClick() {
+	$('.js-dropdown-btn-classes').click(function(e) {
+		e.preventDefault();
+
+		$('.js-actions-dropdown-classes').toggleClass('show');
+	});
+
+	window.onclick = function(e) {
```

Comment:
Should not need if we bind the above with "on":
$('.js-dropdown-btn-classes').on('click',function(e)....)